ui/desktop/app/routes/scopes/scope/projects.js

```diff
@@ -99,13 +97,15 @@ export default class ScopesScopeProjectsRoute extends Route {
             token,
           });
           this.job.start();
+          __electronLog?.info('Starting polling of new sessions again');
           return;
         } catch (e) {
-          // TODO: Log this error
+          __electronLog?.error('Failed to add token to daemons', e);
```

Comment:
nitpick: is the message supposed to be not plural ?
```suggestion
          __electronLog?.error('Failed to add token to daemon', e);
```

Reply:
It actually is meant to be plural bc right now it will add tokens to both cache daemon and the client agent daemon, even though we really care about one at this point in time

Reply:
ahh gotcha that makes sense, thank you!